### PR TITLE
Added my YuvrajSingh.md to Readmes

### PR DESCRIPTION
**Pull request by Yuvraj883**

### CODE REVIEW

File: Readmes/YuvrajSingh.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+- 💻 Languages and Tools I use:   <code><img height="20" src="https://raw.githubusercontent.com/github/explore/80688e429a7d4ef2fca1e82350fe8e3517d3494d/topics/visual-studio-code/visual-studio-code.png"></code>
+<code><img height="20" src="https://raw.githubusercontent.com/github/explore/80688e429a7d4ef2fca1e82350fe8e3517d3494d/topics/javascript/javascript.png"></code>
+<code><img height="20" src="https://raw.githubusercontent.com/github/explore/80688e429a7d4ef2fca1e82350fe8e3517d3494d/topics/python/python.png"></code>
+<code><img height="20" src="https://raw.githubusercontent.com/github/explore/80688e429a7d4ef2fca1e82350fe8e3517d3494d/topics/cpp/cpp.png"></code>
+<code><img height = "20" src = "https://raw.githubusercontent.com/github/explore/80688e429a7d4ef2fca1e82350fe8e3517d3494d/topics/html/html.png"></code>
+<code><img height = "20" src = "https://raw.githubusercontent.com/github/explore/80688e429a7d4ef2fca1e82350fe8e3517d3494d/topics/css/css.png"></code>
+<code><img height="20" src="https://raw.githubusercontent.com/github/explore/80688e429a7d4ef2fca1e82350fe8e3517d3494d/topics/git/git.png"></code>
+
+
+📫 How to reach me:
```
Comment (Panquesito7) [inline Readmes/YuvrajSingh.md:33]:
```suggestion
📫 How to reach me
```

File: Readmes/YuvrajSingh.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+<p align="center"> 
+  Visitor count<br>
+  <img src="https://profile-counter.glitch.me/Mohasin-Haque/count.svg" />
+</p>
+ <div align="center">
+  
+  [![Spotify playing](http://spotify.aio-api.ml/spotify?id=qy9jhr85so9g8pr6zz7aizc6x&theme=wavy&image=true&bars_when_not_listening=true&bg_color=black&title_color=cyan&text_color=cyan)](https://open.spotify.com/user/qy9jhr85so9g8pr6zz7aizc6x)
+ 
+ ### Show some ❤️ by starring some of the repositories!
```
Comment (Panquesito7) [inline Readmes/YuvrajSingh.md:68]:
```suggestion
### Show some ❤️ by starring some of the repositories!
```

File: Readmes/YuvrajSingh.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  </p> 
+   <p align='center'>
+  <img width="450px" src="https://github-readme-stats.vercel.app/api?username=Mohasin-Haque&count_private=true&theme=radical"/>
+</p>
+  <p align='center'>
+  <img src = "https://github-readme-stats.vercel.app/api/top-langs/?username=Mohasin-Haque&theme=radical&hide=jupyter%20notebook&layout=compact&langs_count=8"></p>
+
+<p align="center"> 
+  Visitor count<br>
+  <img src="https://profile-counter.glitch.me/Mohasin-Haque/count.svg" />
```
Comment (Panquesito7) [inline Readmes/YuvrajSingh.md:62]: I recommend removing the visitor count badge as it's inaccurate.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by Greeshma2903** [comment]
Hey @Yuvraj883, please let us know if you want to make the changes requested. We'll be able to go ahead with the merge then.

**Comment by Greeshma2903** [comment]
Closing #45 due to inactivity

**Pull request closed without merging**